### DISCUSSION TITLE
How to hide always some tables on ER Diagram

### DISCUSSION THREAD

**Discussion by ghost** [dbeaver/dbeaver#9033] [Q&A] [answered]
- Operating system (distribution) and version: W10 Pro
- DBeaver version: 7.0.4

I do a WP plugin that creates a some new tables on wp's database. This tables have own relations with FKs AND pks, not like WP database :/. After update my tables, when i try to view my ER Diagram i always has to remove the WP tables that i don't want to see. Any solution or config?

**Comment by kseniiaguzeeva** [comment, marked as answer]
Have you tried to create your custom diagram?

1. Press File>New>Create diagram>Next
2. Select tables you want and write a diagram name
3. Press Finish

You can delete any tables from the custom diagram
![dbeaver_Lg9QweS4K9](https://user-images.githubusercontent.com/51405061/85275954-bea19080-b489-11ea-8348-bdfe63d0a2b6.png)

**Comment by ghost** [comment]
No, i use the default ER diagram for the database.
This works for me. Thanks!!
